src/application/engine/rendering/ClientMap.java

```diff
@@ -253,9 +280,12 @@ public void updateScores(HashMap<Integer, Double> scoreMap)
     {
         for (Integer id : scoreMap.keySet())
         {
+
             GameObject go = gameObjects.get(id);
-            if (go != null)
+
+            if (go != null && go instanceof BoDCharacter)
```

Comment:
we actually want an exception if the cast fails, thus we should not use instanceof here